httpmw/recovery/recovery_test.go

```diff
@@ -0,0 +1,65 @@
+package recovery
+
+import (
+	"fmt"
+	"log/slog"
+	"net/http"
+	"net/http/httptest"
+	"os"
+
+	"github.com/Azure/aks-middleware/httpmw/logging"
+	"github.com/gorilla/mux"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Httpmw", func() {
+	var (
+		router *mux.Router
+	)
+
+	BeforeEach(func() {
+		router = mux.NewRouter()
+	})
+
+	Describe("PanicMiddleware", func() {
+		It("should handle panic and return Internal Server Error", func() {
+			logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
+			router.Use(NewPanicHandling(logger, customPanicHandler))
+			router.HandleFunc("/", func(w http.ResponseWriter, e *http.Request) {
+				panic("oops")
+			})
+
+			rw := httptest.NewRecorder()
+			req := httptest.NewRequest("GET", "/", nil)
+
+			router.ServeHTTP(rw, req)
+
+			Expect(rw.Body.String()).To(ContainSubstring("Internal Server Error"))
```

Comment:
Change the expectation to differentiate it from the default handler.

Reply:
done